packages/grid/_modules_/grid/components/columnSelection/GridHeaderCheckbox.tsx

```diff
@@ -103,7 +103,7 @@ const GridHeaderCheckbox = React.forwardRef<HTMLInputElement, GridColumnHeaderPa
           event.stopPropagation();
         }
         if (isNavigationKey(event.key) && !event.shiftKey) {
```

Comment:
Shouldn't we fire the `columnHeaderKeyDown` on any key, not just navigation one ?

Reply:
Should we remove this handler and let `columnHeaderKeyDown` to propagate freely?